tests/unit/test_attribute_collection.py

```diff
@@ -0,0 +1,64 @@
+from unittest import TestCase
```

Comment:
This is a duplicate of tests/unit/test_attribute_configuration.py and seems misnamed